### PR TITLE
remove  service from update payment as orchestrator can not provide

### PR DESCRIPTION
**Pull request by mdayican**
### JIRA link (if applicable) ###

https://tools.hmcts.net/jira/browse/BPS-766

### Change description ###

remove  service from update payment model as orchestrator can not provide

**Does this PR introduce a breaking change?** (check one with "x")

```
[ ] Yes
[x] No
```

### CODE REVIEW

File: src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/model/UpdatePaymentMessage.java [hunk 1]
```diff
@@ -6,20 +6,17 @@
...
-    public final String service;
     public final String exceptionRecordRef;
     public final String newCaseRef;
 
     public UpdatePaymentMessage(
-        @JsonProperty(value = "envelope_id", required = true) String envelopeId,
-        @JsonProperty(value = "jurisdiction", required = true) String jurisdiction,
-        @JsonProperty(value = "service", required = true) String service,
+        @JsonProperty(value = "envelope_id") String envelopeId,
+        @JsonProperty(value = "jurisdiction") String jurisdiction,
```
Comment (lgonczar) [inline src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/model/UpdatePaymentMessage.java]: Are these two fields not required anymore?

Comment (rkondratowicz) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/model/UpdatePaymentMessage.java]: @lgonczar only `service` was removed. It was only used for logging

Comment (lgonczar) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/model/UpdatePaymentMessage.java]: But now `envelope_id` and `jurisdiction` are not required. Was it intended?

Comment (mdayican) [reply src/main/java/uk/gov/hmcts/reform/bulkscan/payment/processor/service/servicebus/model/UpdatePaymentMessage.java]: thought without envelope id and jurisdiction we should process the payment.  so made them optional

**Comment by mdayican** [comment]
will recreate a new pr as it is stuck.

**Comment by mdayican** [comment]
will recreate a new pr as it is stuck.

**Pull request closed without merging**